kura/examples/bnd_examples/org.eclipse.kura.demo.bnd.heater/pom.xml

```diff
@@ -20,7 +20,7 @@
 
     <groupId>org.eclipse.kura</groupId>
     <artifactId>org.eclipse.kura.demo.bnd.heater</artifactId>
-    <version>1.1.200-SNAPSHOT</version>
+    <version>1.1.200</version>
```

Comment:
This version seems to use the old schema. Probably it was not properly updated in develop. I think it should be 1.2.0. What do you think?